homework/v.paslavskiy/Hometask_5.py

```diff
@@ -0,0 +1,16 @@
+def bubble_sort(lst):
```

Comment:
in general Approved. 
Extra task: 
- add key function as function argument, which will compare two items. default function use `lambda x, y: x>y`
- add reverse parameter to function.